### DISCUSSION TITLE
Remove <body> tag from getHTML()

### DISCUSSION THREAD

**Discussion by CNSam** [GrapesJS/grapesjs#4499] [Q&A]
Hello,

I've just started implementing GrapeJS and all is going well so far except for when I use the getHTML() method.

In short it is also providing the default <body> tags which I could do without. I was wondering if there is a way to either change these to a DIV or remove them from the output altogether but keep the remaining tags/components?

TIA

**Comment by artf** [comment]
You can update the wrapper component in this way
```js
editor.Components.addType('wrapper', {
 model: {
   defaults: {
     tagName: 'div', // use `div` instead of `body` in HTML export...
   },

   // ...or skip the wrapper tag
   toHTML(opts) {
     return this.getInnerHTML(opts);
   }
 }
})
```

**Reply by krishnaeverestengineering** [reply]
This code should work only when overriding the wrapper from plugin. Plugins will load before rendering the editor.

```
      grapesjs.plugins.add('test', function(editor, opts = {}) {
        editor.Components.addType('wrapper', {
          model: {
            defaults: {
              tagName: 'div', // use `div` instead of `body` in HTML export
            },
        
            // Skip wrapper in the HTML output
            toHTML(opts) {
              console.log("Heck yah")
              return this.getInnerHTML(opts);
            }
          }
        })
      })
```

**Reply by artf** [reply]
As per [documentation](https://grapesjs.com/docs/modules/Components.html#define-custom-component-type), all custom components have to be placed in a plugin.

**Comment by ronaldohoch** [comment]
Why not this?

`editor.getHtml().replace("<body>","").replace("</body>","")`
